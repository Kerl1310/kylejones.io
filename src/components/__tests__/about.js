import React from 'react'
import renderer from 'react-test-renderer'

import About from '../about'

describe('About', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<About title="my-title" text="my-text" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
