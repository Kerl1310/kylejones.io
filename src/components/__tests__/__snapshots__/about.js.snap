// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders correctly 1`] = `
<div>
  <h1>
    About
  </h1>
  <p
    dangerouslySetInnerHTML={
      Object {
        "__html": "my-text",
      }
    }
  />
</div>
`;
